.changeset/afraid-pumas-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': major
```

Comment:
This should not be a major release!! 😅

Reply:
Oh my, didn't notice that! Will make the changes ASAP